### ISSUE TITLE
Content does not load...

### ISSUE THREAD

**Issue by Rizora** [niazoff/Staff#1]
Crashes upon click no error message

**Comment by niazoff** [comment]
Hi 👋 thanks for opening the issue. I haven’t really maintained the repo after I had my interview at Vimeo. Are you trying to use Vimeo’s API and need help?

**Comment by Rizora** [comment]
Yes, I'm trying to find a way to load an album into a view and allow users to watch natively.
